core/commands/partials/vagrant_configurator.rb

```diff
@@ -198,13 +186,14 @@ def up_node(node)
   def up
     nodes = @config.node_names
     run_in_directory(@config.path) do
-      store_network_config
+      @network_config = ConfigurationFileManager.store_network_config(@config, @ui)
       up_results = Workers.map(nodes) { |node| up_node(node) }
       up_results.each { |up_result| up_result[1].print_to_stdout } if use_log_storage?
       return ERROR_RESULT unless up_results.detect { |up_result| !up_result[0] }.nil?
+
     end
-    generate_config_information(Dir.pwd)
-    generate_label_information_file
+    ConfigurationFileManager.generate_config_information(@config, @network_config, Dir.pwd, @ui)
```

Comment:
You call these methods at the same time. Why make two of them and call them one-by-one? Change it to the one method instead.